bitnami/harbor/templates/exporter/exporter-dpl.yaml

```diff
@@ -0,0 +1,184 @@
+            {{- include "common.tplvalues.render" (dict "value" .Values.exporter.extraEnvVars "context" $) | nindent 12 }}
+            {{- end }}
+          envFrom:
+            - configMapRef:
+                name: "{{ include "harbor.exporter" . }}-envvars"
+          ports:
+            - containerPort: {{ .Values.exporter.containerPorts.metrics }}
+              name: {{ include "harbor.metricsPortName" . }}
+          {{- if not .Values.diagnosticMode.enabled }}
+          {{- if .Values.exporter.startupProbe.enabled }}
+          startupProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.exporter.startupProbe "enabled") "context" $) | nindent 12 }}
+            httpGet:
+              path: {{ .Values.metrics.path | quote }}
+              port: {{ include "harbor.metricsPortName" . }}
+              # Metrics are exposed only though HTTP https://github.com/goharbor/harbor/issues/16252
+              scheme: HTTP
+          {{- else if .Values.exporter.customStartupProbe }}
+          startupProbe: {{- include "common.tplvalues.render" (dict "value" .Values.exporter.customStartupProbe "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.exporter.livenessProbe.enabled }}
+          livenessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.exporter.livenessProbe "enabled") "context" $) | nindent 12 }}
+            httpGet:
+              path: {{ .Values.metrics.path | quote }}
+              port: {{ include "harbor.metricsPortName" . }}
+              scheme: HTTP
+          {{- else if .Values.exporter.customLivenessProbe }}
+          livenessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.exporter.customLivenessProbe "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.exporter.readinessProbe.enabled }}
+          readinessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.exporter.readinessProbe "enabled") "context" $) | nindent 12 }}
+            httpGet:
+              path: {{ .Values.metrics.path | quote }}
+              port: {{ include "harbor.metricsPortName" . }}
+              scheme: HTTP
+          {{- else if .Values.exporter.customReadinessProbe }}
+          readinessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.exporter.customReadinessProbe "context" $) | nindent 12 }}
+          {{- end }}
+          {{- end }}
+          {{- if .Values.exporter.lifecycleHooks }}
+          lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.exporter.lifecycleHooks "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.exporter.resources }}
+          resources: {{- toYaml .Values.exporter.resources | nindent 12 }}
+          {{- end }}
+          volumeMounts:
+            {{- if .Values.internalTLS.enabled }}
+            - name: internal-tls-certs
+              mountPath: /etc/harbor/ssl/exporter
+            {{- end }}
```

Comment:
is the exporter really taken into account these certs?

Reply:
For internal connections to harbor-core. Only CA.crt certificate is needed